coalib/parsing/LineParser.py

```diff
@@ -57,17 +59,24 @@ def parse(self, line):
             self.comment_separators)
         comment = unescape(comment)
         if line == '':
-            return '', [], '', comment
+            return '', [], '', False, comment
```

Comment:
docstring not updated.
As a breaking change, if you think this should be safe to change without deprecation, the  function name should be preceeded with _ so that it is clear in the future that it may change without warning.